__tests__/users/App.test.js

```diff
@@ -94,9 +94,7 @@ describe('App Component', () => {
 
   it('should update the URL query string and re-render the app', async () => {
     await page.waitForSelector('[data_key="verified"]');
-
-    // Click on the "Linked Accounts" tab
-    await page.click('[data_key="verified"]');
+    await page.select('.tabs_section', 'verified');
```

Comment:
Instead of class use data test id

Reply:
> Instead of class use data test id

Hey Vinit, I don't understand what is the 'data test id' you want me to use? do I have to add a seperate property 'data-testid' in the TabsSection.js file for it to be used in the test files?

Reply:
> Instead of class use data test id

updated with data test id, please review it once